tests/integration/reports/multi_table/_properties/test_relationship_validity.py

```diff
@@ -0,0 +1,39 @@
+from unittest.mock import Mock
+
+from tqdm import tqdm
+
+from sdmetrics.demos import load_demo
+from sdmetrics.reports.multi_table._properties import RelationshipValidity
+
+
+class TestRelationshipValidity:
+
+    def test_end_to_end(self):
+        """Test the ``RelationshipValidity`` multi-table property end to end."""
+        # Setup
+        real_data, synthetic_data, metadata = load_demo(modality='multi_table')
+        relationship_validity = RelationshipValidity()
+
+        # Run
+        result = relationship_validity.get_score(real_data, synthetic_data, metadata)
+
+        # Assert
+        assert result == 1.0
+
+    def test_with_progress_bar(self):
+        """Test that the progress bar is correctly updated."""
+        # Setup
+        real_data, synthetic_data, metadata = load_demo(modality='multi_table')
+        relationship_validity = RelationshipValidity()
+        num_relationship = 2
+
+        progress_bar = tqdm(total=num_relationship)
+        mock_update = Mock()
```

Comment:
Instead of mocking I think we can use a fixture like [capsys](https://docs.pytest.org/en/7.4.x/how-to/capture-stdout-stderr.html) and try to evaluate the actual printed output

Reply:
Yes, done in [a5afedd](https://github.com/sdv-dev/SDMetrics/pull/488/commits/a5afeddfa288b600e992ca10294b44f622cec736)